### ISSUE TITLE
Failing to reproduce IRR-PWC results on KITTI

### ISSUE THREAD

**Issue by larjung** [visinf/irr#19]
Hi,
First of all I'd like to thank you so much for sharing this great repo.

As a sanity check, I am trying to reproduce your results on KITTI by finetuning from things3d, using the checkpoint of "IRR-PWC_things3d" that you supply, but so far without success.
I ran your script "IRR-PWC_kitti_train.sh" which does exactly that , without changing any parameters, and all the training ran smoothly and finished without any errors.
However, when I am evaluating the newly created checkpoint on KITTI and comparing it with your own supplied checkpoint (IRR-PWC_kitti), based on your evaluation script in "IRR-PWC_kitti_train.sh", I get completely different results:
- Yours (from "saved_check_point/pwcnet/IRR-PWC_kitti/checkpoint_latest.ckpt"): epe_avg=1.33, outlier_avg=0.0421.
- Mine (after running your finetune script as-is): epe_avg=2.517, outlier_avg=0.0856 (almost double!)
- Reference (from "saved_check_point/pwcnet/IRR-PWC_things3d/checkpoint_latest.ckpt"): epe_avg=2.552, outlier_avg=0.0864 

So my reproduced error on KITTI is two times worse than your own checkpoint. And even worse, I got almost same error after finetune and before finetune, which means that this finetune did nothing practically.

Could you think of anything that could explain this strange behavior, and my inability to reproduce your results, using your scripts?

Thank you,
Danny

**Comment by hurjunhwa** [comment]
Hi,

We used two-stage finetuning strategies.
First, we used "IRR-PWC_kitti_train.sh" to find a stopping point (or the number of iteration steps) using Train/Valid split.
And then given the number of iteration steps, we used "IRR-PWC_kitti_train_full.sh" to train our model using all KITTI images.

So, the provided fine-tuned model uses the validation images for the training, which is not included when running the script, "IRR-PWC_kitti_train.sh".
That could be the reason why the accuracy differs.

Or, I recently changed the implementation and reverted back again.
If you downloaded the code in the middle, you may have a different implementation.
Maybe pulling the current one again would solve the problem if this is the case.

Thanks,
Jun

**Comment by larjung** [comment]
Hi Jun,
Thanks a lot for your answer! I can now successfully reproduce your results.

All the best,
Danny
